src/abbfreeathome/devices/heating_actuator.py

```diff
@@ -0,0 +1,90 @@
+"""Free@Home HeatingActuator Class."""
+
+from typing import Any
+
+from ..api import FreeAtHomeApi
+from ..bin.pairing import Pairing
+from .base import Base
+
+
+class HeatingActuator(Base):
+    """Free@Home HeatingActuator Class."""
+
+    _state_refresh_output_pairings: list[Pairing] = [
+        Pairing.AL_INFO_VALUE_HEATING,
+    ]
+
+    def __init__(
+        self,
+        device_id: str,
+        device_name: str,
+        channel_id: str,
+        channel_name: str,
+        inputs: dict[str, dict[str, Any]],
+        outputs: dict[str, dict[str, Any]],
+        parameters: dict[str, dict[str, Any]],
+        api: FreeAtHomeApi,
+        floor_name: str | None = None,
+        room_name: str | None = None,
+    ) -> None:
+        """Initialize the Free@Home HeatingActuator class."""
+        self._state: int | None = None
+
+        super().__init__(
+            device_id,
+            device_name,
+            channel_id,
+            channel_name,
+            inputs,
+            outputs,
+            parameters,
+            api,
+            floor_name,
+            room_name,
+        )
+
+    @property
+    def state(self) -> int | None:
+        """Get the state of the actuator."""
+        return self._state
+
+    async def set_state(self, value: int):
+        """
+        Set the open level of the heating valve.
+
+        The position has to be between 0 and 100
+        Fully closed = 0
+        Fully open = 100
+        Just as an information: This is exaclty the other way round as done in HA,
```

Comment:
Since this is just a valve I was thinking it should use the Valve [platform](https://developers.home-assistant.io/docs/core/entity/valve) in Home Assistant.

The Valve platform operates the same as this actuator where 0 means closed and 100 means fully open.

Was there another platform this might line up with?

Reply:
You are absolutely right, I totally have overseen that platform.